spec/compressor_spec.rb

```diff
@@ -2,29 +2,6 @@
   describe ".compress" do
     let(:instrumenter) { Kafka::Instrumenter.new(client_id: "test") }
 
-    it "encodes and decodes compressed messages" do
-      compressor = Kafka::Compressor.new(codec_name: :snappy, threshold: 1, instrumenter: instrumenter)
-
-      message1 = Kafka::Protocol::Message.new(value: "hello1")
-      message2 = Kafka::Protocol::Message.new(value: "hello2")
-
-      message_set = Kafka::Protocol::MessageSet.new(messages: [message1, message2])
-      compressed_message_set = compressor.compress(message_set)
-
-      data = Kafka::Protocol::Encoder.encode_with(compressed_message_set)
-      decoder = Kafka::Protocol::Decoder.from_string(data)
-      decoded_message = Kafka::Protocol::Message.decode(decoder)
-      decoded_message_set = decoded_message.decompress
-      messages = decoded_message_set.messages
-
-      expect(messages.map(&:value)).to eq ["hello1", "hello2"]
-
-      # When decoding a compressed message, the offsets are calculated relative to that
-      # of the container message. The broker will set the offset in normal operation,
-      # but at the client-side we set it to -1.
-      expect(messages.map(&:offset)).to eq [-1, 0]
-    end
```

Comment:
Why remove this?

Reply:
We moved tests to message_set_spec, and we beleived this case was already covered in the other three tests we provided. This test does has -1 as wrapper message offset, and both messages are -1, -1 as well. Hence our code will accept this as "correct from broker side" and return `[-1, -1]`

In the end we just thought the other test cases was more realistic. Should we add this back?

Reply:
OK – no, that's fine, just needed to follow along.